docs/ja/user-guide/configuration/sourceDirectory.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: main
+title: ソースディレクトリ
+category: User Guide
+menu: menu
```

Comment:
The same comment of https://github.com/screwdriver-cd/guide/pull/314/files#r296506881 .

Reply:
fixed~